test/python/WMComponent_t/AnalyticsDataCollector_t/AnalyticsDataCollector_t.py

```diff
@@ -76,8 +71,8 @@ def getConfig(self):
 
         General config file
         """
-        #configPath=os.path.join(WMCore.WMInit.getWMBASE(), \
-        #                        'src/python/WMComponent/WorkQueueManager/DefaultConfig.py')):
+        # configPath=os.path.join(WMCore.WMInit.getWMBASE(), \
+        #                         'src/python/WMComponent/WorkQueueManager/DefaultConfig.py')):
 
```

Comment:
May be we can just remove these 2 lines. since it is not used.

Reply:
Thanks @ticoann , I just removed these lines.